### PR TITLE
Add compatibility with TYPO3 9.5 and drop compatibility with 8.7

### PR DESCRIPTION
**Pull request by eliashaeussler**
The extension "persons" has been updated to be compatible with TYPO3 9.5. As this version forces some deprecations, the affected code has been rewritten and is no longer compatible with TYPO3 8.7.

### CODE REVIEW

File: ext_tables.sql [hunk 1]
```diff
@@ -27,34 +23,10 @@ CREATE TABLE tx_persons_domain_model_person (
 	status int(11) unsigned DEFAULT '0' NOT NULL ,
 	categories int(11) UNSIGNED DEFAULT '0' NOT NULL,
 
-	tstamp int(11) unsigned DEFAULT '0' NOT NULL,
```
Comment (dwenzel) [inline ext_tables.sql:30]: Why did you delete all the boilerplate fields?

Comment (eliashaeussler) [reply ext_tables.sql:30]: Since 9.3, it's no longer necessary to declare these fields in `ext_tables.sql` as the database schema analyzer automatically creates TYPO3 “management” related database columns out of the declared TCA. That's the reason why I removed all those fields.

https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.3/Feature-85160-AutoCreateManagementDBFieldsFromTCACtrl.html

Comment (dwenzel) [reply ext_tables.sql:30]: great

**Comment by eliashaeussler** [comment]
I have rewritten the unit tests in the form that assertions are no longer made against private class attributes. From my perspective, this PR is now ready to be merged.
